src/modules/FileLocksmith/FileLocksmithUI/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace PowerToys.FileLocksmithUI.ViewModels
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using CommunityToolkit.Mvvm.ComponentModel;
+    using CommunityToolkit.Mvvm.Input;
+    using FileLocksmith.Interop;
+    using global::FileLocksmithUI;
+    using global::FileLocksmithUI.Helpers;
+    using Microsoft.UI.Dispatching;
+    using Microsoft.UI.Xaml.Controls;
+
+#pragma warning disable CA1708 // Identifiers should differ by more than case
+    public partial class MainViewModel : ObservableObject, IDisposable
+#pragma warning restore CA1708 // Identifiers should differ by more than case
+    {
+        public IAsyncRelayCommand LoadProcessesCommand { get; }
+
+        private bool _isLoading;
+        private bool _isElevated;
+        private string[] paths;
+        private bool _disposed;
+
+        public ObservableCollection<ProcessResult> Processes { get; } = new ();
+
+        public bool IsLoading
+        {
+            get
+            {
+                return _isLoading;
+            }
+
+            set
+            {
+                _isLoading = value;
+                OnPropertyChanged(nameof(IsLoading));
+            }
+        }
+
+        public bool IsElevated
+        {
+            get
+            {
+                return _isElevated;
+            }
+
+            set
+            {
+                _isElevated = value;
+                OnPropertyChanged(nameof(IsElevated));
+            }
+        }
+
+        public string[] Paths
+        {
+            get => paths;
+            set
+            {
+                paths = value;
+                OnPropertyChanged(nameof(Paths));
+            }
+        }
+
+        public string PathsToString
+        {
+            get
+            {
+                return string.Join("; ", paths);
```

Comment:
```suggestion
                return string.Join(";\n", paths);
```

Using a line break instead of a space makes reading the paths easier.
![image](https://user-images.githubusercontent.com/61519853/198382815-815f5f10-8d9a-43ba-832f-b81894679ffc.png)

Reply:
Changed to new line. THank you! ;)